src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -25,19 +25,36 @@ public function testConfigTree($options, $results)
         $processor = new Processor();
         $configuration = new Configuration();
         $config = $processor->processConfiguration($configuration, array($options));
+        // 'position' option is deprecated, but added by Symfony because it has a default value
+        unset($config['position']);
+
+        $this->assertEquals($results, $config);
+    }
+
+    /**
+     * @dataProvider getDebugModes
+     * @group legacy
+     * @expectedDeprecation
+     */
+    public function testConfigTreeWithDeprecatedPosition($options, $results)
+    {
+        $results['position'] = 'bottom';
```

Comment:
you should also put different values in the options, to ensure that the deprecated option can be used.

Reply:
indeed, I thought there was only one occurrence. All existing tests should be kept as legacy